File: Backend/DisasterDispatch.Core/Dtos/OperationEmployeeDtos/OperationEmployeeDto.cs
using DisasterDispatch.Core.Dtos.CustomOperationDtos;
using DisasterDispatch.Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DisasterDispatch.Core.Dtos.OperationEmployeeDtos
{
    public class OperationEmployeeDto
    {
        public Guid Id { get; set; }
        public Guid CustomOperationId { get; set; }
        public CustomOperationDto CustomOperation{ get; set; }
        public string AppUserId { get; set; }
        public AppUser AppUser{ get; set; }
    }
}

File: Frontend/DisasterDispatchFrontend/src/app/Models/DTOS/OperationEmployee/operation-employee-dto.ts
import { Guid } from "guid-typescript";
import { CustomOperation } from "../../custom-operation";
import { AppUser } from "../../app-user";

export class OperationEmployeeDto {
    public id: Guid;
    public customOperationId: Guid;
    public customOperation: CustomOperation | null;
    public appUserId: string;
    public appUser: AppUser | null;
}
